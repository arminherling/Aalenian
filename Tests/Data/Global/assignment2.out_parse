AssignmentStatement: {
    Left: {
        NameExpression: a
    }
    Right: {
        FunctionCallExpression: {
            NameExpression: func
            ArgumentsNode(0): {
            }
        }
    }
}
