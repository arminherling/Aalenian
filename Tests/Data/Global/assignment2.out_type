TypedAssignmentStatement: {
    Type: ???
    Left: {
        TypedConstant: {
            Type: ???
            Name: a
        }
    }
    Right: {
        TypedFunctionCallExpression: {
            Type: ???
            Name: func
            ArgumentsNode(0): {
            }
        }
    }
}
