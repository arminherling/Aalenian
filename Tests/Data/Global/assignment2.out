AssignmentStatement: {
    Left: {
        Name: a
    }
    Right: {
        FunctionCall: {
            Name: func
            Arguments(0): {
            }
        }
    }
}
